client/src/main.rs

```diff
@@ -1059,6 +1059,68 @@ fn main() {
                     Ok(())
                 }),
         )
+        .add_cmd(
+                Command::new("purge-community-ceremony")
+                    .description("purge all history within the provided ceremony index range for the specified community")
+                    .options(|app| {
+                        app.setting(AppSettings::ColoredHelp)
+                            .from_cindex_arg()
+                            .to_cindex_arg()
+
+                    })
+                .runner(|_args: &str, matches: &ArgMatches<'_>| {
+                    let sudoer = AccountKeyring::Alice.pair();
+                    let api = get_chain_api(matches).set_signer(sudoer);
+
+                    let current_ceremony_index = get_ceremony_index(&api);
+
+                    let from_cindex_arg = matches.from_cindex_arg().unwrap_or(0);
+                    let to_cindex_arg = matches.to_cindex_arg().unwrap_or(0);
+
+                    let from_cindex = into_effective_cindex(from_cindex_arg, current_ceremony_index);
+                    let to_cindex = into_effective_cindex(to_cindex_arg, current_ceremony_index);
+
+                    let cid = verify_cid(&api,
+                                         matches
+                                             .cid_arg()
+                                             .expect("please supply argument --cid"),
+                    );
+                    println!("purging ceremony index range [{}  {}] for community {}", from_cindex, to_cindex, cid);
+
+                    let calls: Vec<_> = (from_cindex..to_cindex+1)
+                        .map(|idx| compose_call!(
```

Comment:
I think this should be possible and more intuitive: `(from_cindex..=to_cindex)`

Reply:
true